editors/code/package.json

```diff
@@ -460,10 +460,10 @@
                     "default": true,
                     "markdownDescription": "Whether to show postfix snippets like `dbg`, `if`, `not`, etc."
                 },
-                "rust-analyzer.completion.enableExperimental": {
+                "rust-analyzer.completion.disableFuzzyAutoimports": {
```

Comment:
Let's phrase this as `"thing.enable" = true` for consistency.

Reply:
r=me with setting name fixed!

Reply:
One concern: would it be confusing to find this setting on, but still not working? 
It might be true for a while, that the clients won't send the completion capability we expect and we'll turn the feature off.

This inversion saves us from the confusion.

Reply:
I think internal consistency of settings is more important -- it's a pain to rename options, and, unless you are really strict about the style, they detorieate.